photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NTDataPublisher.java

```diff
@@ -130,15 +132,25 @@ public void updateCameraNickname(String newCameraNickname) {
 
     @Override
     public void accept(CVPipelineResult result) {
+        CVPipelineResult acceptedResult;
+        if (result instanceof CalibrationPipelineResult)
+            acceptedResult =
```

Comment:
Why not just return early here?

Reply:
The thought was that we would continue to post info to network tables during the calibration. Basically zero out all the 
 data.

Reply:
I'd argue it's expected that the camera not produce info on NT during calibration

Reply:
I figured that we would keep the data on nt up to date and if we can write that the data is not up to date (by replacing it) then we should but if you think not whatever.

Reply:
Yeah fair enough. Just drop a comment in code explaining why we do this